velox/common/encode/Base64.cpp

```diff
@@ -132,25 +143,27 @@ static_assert(
         kBase64Charset,
         kBase64ReverseIndexTable),
     "kBase64ReverseIndexTable has incorrect entries.");
+
 // Verify that for every entry in kBase64ReverseIndexTable, the corresponding
 // entry in kBase64Charset is correct.
-// We can't run this check as the URL version has two duplicate entries so that
-// the url decoder can handle url encodings and default encodings
+// We can't run this check as the URL version has two duplicate entries so
+// that the url decoder can handle url encodings and default encodings
 // static_assert(
 //     checkReverseIndex(
 //         sizeof(kBase64UrlReverseIndexTable) - 1,
 //         kBase64UrlCharset,
 //         kBase64UrlReverseIndexTable),
 //     "kBase64UrlReverseIndexTable has incorrect entries.");
 
+// Implementation of Base64 encoding and decoding functions.
 template <class T>
-/*  static */ std::string
-Base64::encodeImpl(const T& data, const Charset& charset, bool include_pad) {
+/* static */ std::string Base64::encodeImpl(
```

Comment:
// static

Reply:
Fixed in https://github.com/facebookincubator/velox/pull/11299